WatchDogCore/WatchDog/src/nl/tudelft/watchdog/core/logic/event/eventtypes/EventBase.java

```diff
@@ -7,7 +7,7 @@
 
 import nl.tudelft.watchdog.core.logic.network.WatchDogTransferable;
 
-public abstract class EventBase extends WatchDogTransferable implements Serializable, Comparable<EventBase> {
+public abstract class EventBase extends WatchDogTransferable implements Serializable, Comparable<WatchDogTransferable> {
```

Comment:
I am wondering ... is EventBase a good name if it is used for both Intervals and Events?